docs/src/examples/jump_umap_analysis_with_cosmicqc.py

```diff
@@ -0,0 +1,421 @@
+    [
+        col
+        for col in df_feature_selected_with_cqc_outlier_data.columns.tolist()
+        if "cqc." not in col and "analysis." not in col
+    ]
+]
+# show the modified column count
+len(df_for_normalize_and_feature_select.columns)
+
+# +
+parquet_pycytominer_normalized = "./BR00117012_normalized.parquet"
+
+# check if we already have normalized data
+if not pathlib.Path(parquet_pycytominer_normalized).is_file():
+    # normalize the data using pcytominer
+    df_pycytominer_normalized = pycytominer.normalize(
+        profiles=df_for_normalize_and_feature_select,
+        features="infer",
+        image_features=False,
+        meta_features="infer",
+        method="standardize",
+        output_file=parquet_pycytominer_normalized,
+        output_type="parquet",
+    )
+
+# +
+parquet_pycytominer_feature_selected = "./BR00117012_feature_select.parquet"
+
+# check if we already have feature selected data
+if not pathlib.Path(parquet_pycytominer_feature_selected).is_file():
+    # feature select normalized data using pycytominer
+    df_pycytominer_feature_selected = pycytominer.feature_select(
+        profiles=parquet_pycytominer_normalized,
+        operation=[
+            "variance_threshold",
+            "correlation_threshold",
+            "blocklist",
+            "drop_na_columns",
+        ],
+        na_cutoff=0,
+        output_file=parquet_pycytominer_feature_selected,
+        output_type="parquet",
+    )
+
+
+# +
+def generate_umap_embeddings(
+    df_input: pd.DataFrame, cols_metadata: List[str]
+) -> np.ndarray:
+    """
+    Generates UMAP (Uniform Manifold Approximation and Projection)
+    embeddings for a given input dataframe,
+    excluding specified metadata columns.
+
+    Args:
+        df_input (pd.DataFrame]):
+            A dataframe which is expected to contain
+            numeric columns to be used for UMAP fitting.
+        cols_metadata (List[str]):
+            A list of column names representing
+            metadata columns that should be excluded
+            from the UMAP transformation.
+
+    Returns:
+        np.ndarray:
+            A dataframe containing the UMAP embeddings
+            with 2 components for each row in the input.
+    """
+
+    # Set constants
+    umap_n_components = 2
+
+    # Make sure to reinitialize UMAP instance per plate
+    umap_fit = umap.UMAP(n_components=umap_n_components)
+
+    # Fit UMAP and convert to pandas DataFrame
+    embeddings = umap_fit.fit_transform(
+        X=df_input[
+            [
+                col
+                for col in df_input.columns.tolist()
+                if col not in metadata_cols and "cqc." not in col
+            ]
+        ].select_dtypes(include=[np.number])
+    )
+
+    return embeddings
+
+
+embeddings_with_outliers = generate_umap_embeddings(
+    df_input=pd.read_parquet(parquet_pycytominer_feature_selected),
+    cols_metadata=metadata_cols,
+)
+embeddings_with_outliers
```

Comment:
rather than print everything, I recommend this:

```python
print(embeddings_with_outliers.shape)
embeddings_with_outliers.head(3)
```

It makes things a bit more manageable to render and digest

Reply:
Thanks, will add this in.